test/fixtures/lightscript/named-arrow-functions/skinny-async-gen/actual.js

```diff
@@ -0,0 +1,2 @@
+fn() -/*>
+  yield await x
```

Comment:
these look good – let's also add the following tests:

```
f() =*/>
  yield <- g()
```


```
f() =*/>
  yield <!- g()
```

```
f() =*/>
  x = yield <- g()
```

These tests will also probably need to be copied into the plugin's test suite